packages/run-protocol/src/reserve/assetReserve.js

```diff
@@ -77,20 +92,82 @@ const start = async (zcf, privateArgs) => {
    */
   const addIssuer = async (issuer, keyword) => {
     const brand = await E(issuer).getBrand();
+    trace('addIssuer', { brand, keyword });
     assert(
       keyword !== RunKW && brand !== runKit.brand,
       `${RunKW} is a special case handled by the reserve contract`,
     );
 
-    const liquidityIssuer = E(ammPublicFacet).getLiquidityIssuer(brand);
+    trace('addIssuer storing', {
+      keyword,
+      brand,
+    });
+
+    saveBrandKeyword(brand, keyword);
+    await zcf.saveIssuer(issuer, keyword);
+  };
+
+  /**
+   * @param {Issuer} baseIssuer on which the liquidity issuer is based
+   */
```

Comment:
```suggestion
  /** @param {Issuer} issuer on which the liquidity issuer is based */
```

Reply:
`baseIssuer` isn't part of the sentence. Here's how it looks in an IDE:
<img width="518" alt="Screen Shot 2022-05-27 at 6 41 52 AM" src="https://user-images.githubusercontent.com/21505/170711168-3b5e6fdc-33e2-4648-8f16-2e15745cea5f.png">

If you're proposing to rename the parameter, I think that would make the body less clear. I could add a dash to separate, but I think that's a style decision we should perform universally, not ad hoc in PR review. I could make it `baseIssuer issuer on which` if you think that would be better.

Reply:
stet

Reply:
TIL stet!